### PR TITLE
Buildifier2: address code review comments

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

File: buildifier2/buildifier2.go [hunk 1]
```diff
@@ -27,16 +27,14 @@ func main() {
...
 		if err != nil {
-			fmt.Printf("%+v", err)
-			os.Exit(1)
+			log.Fatalf("%+v\n", err)
 		}
-		newAst := convFile(ast)
+		newAst := convertast.ConvFile(ast)
 		fmt.Print(build.FormatString(newAst))
 
 	default:
```
Comment (laurentlb) [inline buildifier2/buildifier2.go]: You mentioned `log.Info; os.Exit(2)` in the code review, but `log.Fatal` seems simpler. No?

Comment (pmbethe09) [reply buildifier2/buildifier2.go]: log.Fatal produces a nasty stack-trace that is no fun.  So it really shouldn't be used when Exit is available in main

Comment (laurentlb) [reply buildifier2/buildifier2.go]: That's not what I saw when I tried it.
Looking at the doc, I see that: https://golang.org/pkg/log/#Fatal

Comment (pmbethe09) [reply buildifier2/buildifier2.go]: Ah.  I guess I am used to the internal version

**Pull request merged**